app/views/smart_answers/shared/_previous_answers.html.erb

```diff
@@ -1,54 +1,62 @@
 <% if @presenter.current_question_number > 1 %>
-  <table class="govuk-table previous-answers-table">
-    <caption class="govuk-table__caption">
-      <% if local_assigns[:hide_previous_answers] %>
-        <%= render "govuk_publishing_components/components/heading", {
-          text: "Change your answers",
-          heading_level: 2,
-          font_size: "l",
-          margin_bottom: 6
-        } %>
-      <% else %>
-        <%= render "govuk_publishing_components/components/heading", {
-          text: "Your answers",
-          heading_level: 2,
-          font_size: "m",
-          margin_bottom: 4
-        } %>
-      <% end %>
+  <% if local_assigns[:hide_previous_answers] %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Change your answers",
+      heading_level: 2,
+      font_size: "l",
+      margin_bottom: 6
+    } %>
+  <% else %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Your answers",
+      heading_level: 2,
+      font_size: "m",
+      margin_bottom: 4
+    } %>
+  <% end %>
```

Comment:
I wonder if we could make these headings more consistent and simplify the code?

```ruby
<%= render "govuk_publishing_components/components/heading", {
  text: local_assigns[:hide_previous_answers] ? "Change your answers" : "Your answers",
  heading_level: 2,
  font_size: "m",
  margin_bottom: 4
} %>
```

Not sure the reasoning around the difference for margin and font - but from the git history, this was just lifted and shifted from another part of the page, where the larger font and margin may have made more sense. Don't think there is any reason not to make them consistent.

Reply:
Happy to do so! I assumed some design considerations from `find-coronavirus-support` and didn't want to mess with them.

<table>
<tr><th>Before</th><th>After</th></tr>
<tr><td valign="top">

<img width="1265" alt="before" src="https://user-images.githubusercontent.com/788096/117029548-aed41900-acf6-11eb-9098-bc26312d097d.png">


</td><td valign="top">

<img width="1265" alt="after" src="https://user-images.githubusercontent.com/788096/117029556-b09ddc80-acf6-11eb-96b7-2cda03533d5f.png">


</td></tr>
</table>

Reply:
Since the code was already split like this - happy for this to be something we can look at later.